examples/service.yaml

```diff
@@ -16,6 +16,9 @@ node_types:
       network:
         type: string
         description: OpenStack network id (network names are not accepted)
+      security_groups:
```

Comment:
Is there any reason for not using a `list` type here? Using a comma-separated string here feels like a hack.

I would suggest writing this down as:

    security_groups:
      description: OpenStack ...
      type: list
      entry_schema:
        type: string

Reply:
os_server module supports both `list` and `comma-separated string`.  But when I tried  to use `list` type,  xOpera throwed me an exception:
```
Deploying instance model ...
  Processing vm.0 ...
    Executing primary: playbooks/vm/create.yml ...
Traceback (most recent call last):
  File "/home/jst10/opera/.venv/bin/opera", line 11, in <module>
    sys.exit(main())
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/cli.py", line 41, in main
    return args.func(args)
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/commands/deploy.py", line 37, in deploy
    instances.deploy()
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/instances.py", line 142, in deploy
    self.nodes[id].deploy()
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/instances.py", line 61, in deploy
    start=("starting", "started"),
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/instances.py", line 51, in execute_workflow
    success, attributes = operation.run()
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/operations.py", line 18, in run
    k: v.eval(self.instance) for k, v in self.inputs.items()
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/operations.py", line 18, in <dictcomp>
    k: v.eval(self.instance) for k, v in self.inputs.items()
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/types.py", line 188, in eval
    return getattr(reference, function)(*arguments)
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/instances.py", line 72, in get_property
    return self.template.get_property(*path)
  File "/home/jst10/opera/.venv/lib/python3.6/site-packages/opera/types.py", line 521, in get_property
    return prop.eval(self)
AttributeError: 'Pass' object has no attribute 'eval'

```
Because of that I assumed that `list` type is not supported yet in xOpera, so this is reason for using   `comma-separated string`.

Reply:
After looking into code, I think that this:
```
    def eval(self, _reference):
        return self.data
```
is missing from `Pass` class in `opera/types.py:76`.

Reply:
The `Pass` class is a marker for **this is not implemented yet, but I would still like to parse at least something**. If we make it play too nicely with the rest of the code, I might lose interest in fixing bugs and adding new functionality ;)

I will fill the bug on the opera repo and merge this as it is.